RNGoogleMobileAds.podspec

```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.authors             = "Invertase Limited"
   s.source              = { :git => "#{package["repository"]["url"]}.git", :tag => "v#{s.version}" }
   s.social_media_url    = 'http://twitter.com/invertaseio'
-  s.ios.deployment_target = "10.0"
+  s.ios.deployment_target = "12.0"
```

Comment:
Hey @dylancom - as long as you've got a breaking change you might do a 13.0.1 that adds one more break. In order to support app privacy manifests (which will be an Apple requirement shortly), cocoapods has to be at >= 1.12.0

This is how I did it for react-native-firebase, soon to release

```ruby
  s.cocoapods_version   = '>= 1.12.0'
```

https://github.com/invertase/react-native-firebase/pull/7638/commits/2cf4e4d2d9562026a25bbe69868e1ba0056e7a80#diff-14af2ce820d14cb75e1bde06b65a88dd9c151e9054418d2486dc35eaa3c1e69e

Also we may need to document a specific switch to toggle on in their Podfile for `@react-native-firebase/analytics` most people using AdMob will want, from a PR I'm going to merge shortly

```ruby
$RNFirebaseAnalyticsEnableAdSupport = true
```

This links in the AdSupport framework from Apple which Firebase Analytics detects and enables the collection of optional demographic information https://github.com/invertase/react-native-firebase/pull/7637